primitives/types/invalid_transaction_test.go

```diff
@@ -0,0 +1,186 @@
+package types
+
+import (
+	"bytes"
+	"testing"
+
+	sc "github.com/LimeChain/goscale"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_NewInvalidTransactionCall(t *testing.T) {
+	expect := InvalidTransaction{sc.NewVaryingData(InvalidTransactionCall)}
+
+	assert.Equal(t, expect, NewInvalidTransactionCall())
+}
+
+func Test_NewInvalidTransactionPayment(t *testing.T) {
```

Comment:
Generally in the tests, I think that we should be more explicit instead of trying to DRY: 

If I accidentally change the value of `InvalidTransactionPayment`, the test might still pass even though it shouldn't.

`InvalidTransaction{sc.NewVaryingData(sc.U(1))}`
instead of 
`InvalidTransaction{sc.NewVaryingData(InvalidTransactionPayment)}`

Reply:
I don't agree.